text/0000-network-update-loop.md

```diff
@@ -0,0 +1,712 @@
+    PlayerLoop -> NetworkUpdateLoop: RunNetworkPreLateUpdate
+    NetworkUpdateLoop -> NetworkUpdateLoop: UpdateStage = PreLateUpdate
+    loop m_PreLateUpdate_Array
+        NetworkUpdateLoop -> INetworkUpdateSystem: NetworkUpdate
+        NetworkUpdateLoop <-- INetworkUpdateSystem
+    end
+    PlayerLoop <-- NetworkUpdateLoop
+    PlayerLoop -> PlayerLoop: AIUpdatePostScript
+    PlayerLoop -> PlayerLoop: DirectorUpdateAnimationBegin
+    PlayerLoop -> PlayerLoop: LegacyAnimationUpdate
+    PlayerLoop -> PlayerLoop: DirectorUpdateAnimationEnd
+    PlayerLoop -> PlayerLoop: DirectorDeferredEvaluate
+    PlayerLoop -> PlayerLoop: EndGraphicsJobsAfterScriptUpdate
+    PlayerLoop -> PlayerLoop: ConstraintManagerUpdate
+    PlayerLoop -> PlayerLoop: ParticleSystemBeginUpdateAll
+    PlayerLoop -> PlayerLoop: ScriptRunBehaviourLateUpdate
+end
+group PostLateUpdate
+    PlayerLoop -> PlayerLoop: PlayerSendFrameStarted
+    PlayerLoop -> PlayerLoop: DirectorLateUpdate
+    PlayerLoop -> PlayerLoop: ScriptRunDelayedDynamicFrameRate
+    PlayerLoop -> PlayerLoop: PhysicsSkinnedClothBeginUpdate
+    PlayerLoop -> PlayerLoop: UpdateRectTransform
+    PlayerLoop -> PlayerLoop: PlayerUpdateCanvases
+    PlayerLoop -> PlayerLoop: UpdateAudio
+    PlayerLoop -> PlayerLoop: VFXUpdate
+    PlayerLoop -> PlayerLoop: ParticleSystemEndUpdateAll
+    PlayerLoop -> PlayerLoop: EndGraphicsJobsAfterScriptLateUpdate
+    PlayerLoop -> PlayerLoop: UpdateCustomRenderTextures
+    PlayerLoop -> PlayerLoop: UpdateAllRenderers
+    PlayerLoop -> PlayerLoop: EnlightenRuntimeUpdate
+    PlayerLoop -> PlayerLoop: UpdateAllSkinnedMeshes
+    PlayerLoop -> PlayerLoop: ProcessWebSendMessages
+    PlayerLoop -> PlayerLoop: SortingGroupsUpdate
+    PlayerLoop -> PlayerLoop: UpdateVideoTextures
+    PlayerLoop -> PlayerLoop: UpdateVideo
+    PlayerLoop -> PlayerLoop: DirectorRenderImage
+    PlayerLoop -> PlayerLoop: PlayerEmitCanvasGeometry
+    PlayerLoop -> PlayerLoop: PhysicsSkinnedClothFinishUpdate
+    PlayerLoop -> PlayerLoop: FinishFrameRendering
+    PlayerLoop -> PlayerLoop: BatchModeUpdate
+    PlayerLoop -> PlayerLoop: PlayerSendFrameComplete
+    PlayerLoop -> PlayerLoop: UpdateCaptureScreenshot
+    PlayerLoop -> PlayerLoop: PresentAfterDraw
+    PlayerLoop -> PlayerLoop: ClearImmediateRenderers
+    PlayerLoop -> PlayerLoop: PlayerSendFramePostPresent
+    PlayerLoop -> PlayerLoop: UpdateResolution
+    PlayerLoop -> PlayerLoop: InputEndFrame
+    PlayerLoop -> PlayerLoop: TriggerEndOfFrameCallbacks
+    PlayerLoop -> PlayerLoop: GUIClearEvents
+    PlayerLoop -> PlayerLoop: ShaderHandleErrors
+    PlayerLoop -> PlayerLoop: ResetInputAxis
+    PlayerLoop -> PlayerLoop: ThreadedLoadingDebug
+    PlayerLoop -> PlayerLoop: ProfilerSynchronizeStats
+    PlayerLoop -> PlayerLoop: MemoryFrameMaintenance
+    PlayerLoop -> PlayerLoop: ExecuteGameCenterCallbacks
+    PlayerLoop -> PlayerLoop: ProfilerEndFrame
+    PlayerLoop -> NetworkUpdateLoop: RunNetworkPostLateUpdate
```

Comment:
This is too late - after profilerendframe, after memory frame maintenance, after a bunch of graphics internals...